roles/ffh.mesh_wireguard/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: add backports repository
+  apt_repository:
+    repo: deb http://deb.debian.org/debian {{ ansible_distribution_release }}-backports main
+    state: present
+    update_cache: yes
+  when: ' ansible_distribution_release == "buster"'
+
+- name: Install wireguard
+  apt:
+    name: wireguard
+    update_cache: yes
+    cache_valid_time: 3600
+
+- name: Domain configs (supernode)
+  include_tasks: per_domain.yml
+  with_items: "{{domains}}"
+  loop_control:
+    loop_var: domain
+  when: domains is defined and sn is defined
+
+- name: setup batctl service
+  include_tasks: batctl.yml
+
+- name: Firewalling mesh_wireguard for all domains
```

Comment:
Imperative form "Firewall"

Reply:
done in 70d7d06281293d1299b42f14f2f64272acb5f580